utils/build/docker/nodejs/nextjs/next.config.js

```diff
@@ -1,6 +1,6 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-  output: 'standalone'
```

Comment:
can you explain this change ? and should we remove the file if it's empty now ?

Reply:
Standalone mode doesn't work with anything external, so linking dd-trace after the Next build is not possible.

Reply:
Not sure if the file can be removed, I kept it to keep the project structure as close as possible to what the CLI generates (the same is true for a few other files that are probably useless for our purpose).

Reply:
maybe it's worth having 
```javascript
  // disabled because not compatible with linking
  // output: 'standalone'
```